src/components/InvoiceDetails/InvoiceDetails.types.ts

```diff
@@ -0,0 +1,8 @@
+export interface IInvoiceDetailsProps {
+  invoiceData: {
+    id: number;
+    created: string;
```

Comment:
I think it could be Date ?

Reply:
It is already formatted date. I format it in withFormSubmit.tsx in the moment of submitting the form.